packages/translate/lib/testLine.tsx

```diff
@@ -432,6 +437,32 @@ const translateBrowserCommandTestLine = (
 	}
 };
 
+const translateDeviceSettingsTestLine = (
+	testLine: DeviceSettingsTestLine,
+	appConfig?: AppConfiguration,
+	elements?: Elements,
+	lineResult?: TestLineResult,
+): TestLineNode => {
+	const status = testLine.excluded ? 'excluded' : lineResult?.result;
+	const condition = testLine.condition
+		? translateCondition(testLine.condition, false, appConfig, elements, lineResult)
+		: undefined;
+
+	switch (testLine.deviceSettings.type) {
+		case 'setOrientation':
+			const orientation = deviceOrientationsMap[testLine.deviceSettings.params.orientation];
+
+			return <test-line title={
+				<fragment><text>Set screen orientation to</text> <input>{orientation}</input></fragment>
```

Comment:
Do we actually need `<text>` here? I'm not entirely sure, but it should work the same without it:
`<fragment>Set screen orientation <input>{orientation}</input></fragment>`

Reply:
I will check

Reply:
you are right, uodated